android/app/src/main/res/values/colors.xml

```diff
@@ -56,8 +56,8 @@
   <color name="icon_tint">#FF757575</color>
   <color name="icon_tint_night">#FFC4C6C7</color>
 
-  <color name="icon_tint_light">#FFBDBDBD</color>
-  <color name="icon_tint_light_night">#FF76797C</color>
+  <color name="icon_tint_light">#FF007CCB</color>
+  <color name="icon_tint_light_night">#FF2C88AD</color>
```

Comment:
iconTintLight is used in different places, not only for lanes. Won't this change break these other places? If yes, then lanes tint/color should be introduced as a separate constant.

Reply:
fixed